common/com_model.h

```diff
@@ -16,7 +16,7 @@
 #define STUDIO_EVENTS 2
 
 #define MAX_CLIENTS			32
-#define	MAX_EDICTS			900
+#define	MAX_EDICTS			2048
```

Comment:
Is this related?

Reply:
Of course! That depending through how many entities we want to iterate to find triggers!

```cpp
if ((gHUD.m_pShowServerTriggers->value > 0) && (gHUD.m_pShowServerTriggers->value != 2.0f))
	{
		for (int e = 0; e < MAX_EDICTS; ++e)
		{
			cl_entity_t* ent = gEngfuncs.GetEntityByIndex(e);
			if (ent)
			{
```

According to the investigate from Solokiller, max edicts can be no more than 2047, anything higher than this will have cause crash: https://github.com/ValveSoftware/halflife/issues/1777

Reply:
I'd also started to think might be it should controllable instead? Like if the map has a small amonut of triggers, they you wouldn't do extra iterations, right?

Making some cvar like `cl_show_server_triggers_entities_limit 2048`

Reply:
CVar is definitely not the right solution here

Reply:
Let's just leave it as is for now, since it doesn't run with showtriggers 0